deploy/deploy.py

```diff
@@ -167,6 +174,59 @@ def _install_or_update_package(
         )
 
 
+def _grant_sundeck_db_access(cur, sundeck_db: str):
+    print("Running grants to database SUNDECK for application package.")
+    # Grant REFERENCE_USAGE on SUNDECK to the application package
+    # Grant access to the view from sundeck
+    cur.execute(
+        f"""
+    BEGIN
+        CREATE SCHEMA IF NOT EXISTS "{APPLICATION_PACKAGE}".SHARING;
+
+        CREATE DATABASE IF NOT EXISTS "{sundeck_db}";
+        CREATE SCHEMA IF NOT EXISTS "{sundeck_db}".INTERNAL;
+        -- Must be kept in sync with the SUNDECK database
+        CREATE TABLE IF NOT EXISTS "{sundeck_db}".INTERNAL.GLOBAL_QUERY_HISTORY(
+            SNOWFLAKE_ACCOUNT_LOCATOR text,
+            SNOWFLAKE_QUERY_ID text,
+            SUNDECK_QUERY_ID text,
+            FLOW_NAME text,
+            QUERY_TEXT_RECEIVED text,
+            QUERY_TEXT_FINAL text,
+            SNOWFLAKE_SUBMISSION_TIME timestamp_ntz,
+            ALT_WAREHOUSE_ROUTE text,
+            SUNDECK_STATUS text,
+            SUNDECK_ERROR_CODE text,
+            SUNDECK_ERROR_MESSAGE text,
+            SNOWFLAKE_REGION text,
+            SNOWFLAKE_CLOUD text,
+            SUNDECK_START_TIME timestamp_ntz,
+            SUNDECK_ACCOUNT_ID text,
+            ACTIONS_EXECUTED text,
+            SCHEMA_ONLY_REQUEST boolean);
+
+        GRANT REFERENCE_USAGE ON DATABASE {sundeck_db} TO SHARE IN APPLICATION PACKAGE "{APPLICATION_PACKAGE}";
+
+        SHOW REGIONS;
+        CREATE OR REPLACE TABLE "{APPLICATION_PACKAGE}".SHARING.REGIONS(snowflake_region text, cloud text, region text) AS
+            SELECT "snowflake_region", "cloud", "region" from table(result_scan(last_query_id()));
```

Comment:
It looks like we have a disconnect with Azure regions and the region field that Sundeck is exporting. `eastus2` vs. `east-us-2`.

Reply:
I've filed a ticket internally for us and corrected the region data in the view.